docs/UserGuide.md

```diff
@@ -174,26 +174,36 @@ Examples:
 ### Adding a person’s schedule: `addEvent`
 Adds an event to the specified indexed contact.
 
-Format: `addEvent INDEX ed/EVENT_DESCRIPTION da/DATE [ti/TIME] [du/DURATION] [t/TAG]`
+Format: `addEvent INDEX ed/EVENT_DESCRIPTION da/DATE [ti/TIME] [du/DURATION] [r/RECUR_FREQUENCY]`
 * INDEX refers to the index number shown in the displayed person list. The index must be a positive integer 1, 2, …
-* If TIME is not specified, it will be considered as a full-day event.
-* If DURATION is specified, TIME also needs to be specified.
+* If TIME is not specified, it will be considered as a full-day event starting from 00:00.
 * If TIME is specified but not DURATION, the DURATION will be defaulted to 2 hours.
-* DURATION should be in one of the following formats, where X and Y are integer values representing the hours and minutes respectively(not case-sensitive):
+* If DURATION is specified, TIME also needs to be specified.
+* DURATION should be in one of the following formats, where X is a positive integer representing hours, and Y is a positive integer representing minutes(not case-sensitive):
   * XHYM
   * XHY
   * XH
+  * YM
   * X
+  * RECUR_FREQUENCY, if provided, must be one of the following values:
+
+    | Value | Frequency    |
+    |-------|--------------|
+    | `D`   | Content Cell |
+    | `W`   | Content Cell |
```

Comment:
isn't this supposed to be Daily and Weekly?

Reply:
Oh right my bad, I will update it